bodo/src/test/java/org/apache/calcite/test/BodoSqlToRelConverterTest.java

```diff
@@ -99,4 +111,58 @@ public class BodoSqlToRelConverterTest extends SqlToRelTestBase {
         "on value.value = dept2.value";
     sql(sql).ok();
   }
+
+  @Test void testCreateTableOrderBy() {
+    // Tests an error specific to CREATE TABLE with "WITH" and "ORDER BY" clauses
+    // found while doing a test run on tpch Q2
+    final String sql = "CREATE TABLE keaton_testing AS ("
+        +
+        "with part_two as (\n"
+        +
+        "        select 'foo' as p_partkey from (VALUES (1, 2, 3))\n"
+        +
+        "    )\n"
+        +
+        "    select\n"
+        +
+        "                       p_partkey\n"
+        +
+        "                     from\n"
+        +
+        "                       part_two\n"
+        +
+        "                     order by\n"
+        +
+        "                       p_partkey"
+        +
+        ")";
+    sql(sql).ok();
+  }
+
+  @Test void testOrderByNoCreateTable() {
+    // Tests an error specific to CREATE TABLE with "WITH" and "ORDER BY" clauses
```

Comment:
Similar comment to above. What is the error? 
Also tell me what the BE-4483 issue is about in a couple words. Don't paste the link, just do [BE-4483]